spec/services/arcgis_api/geocoder_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'rails_helper'
+
+RSpec.describe ArcgisApi::Geocoder do
+  include ArcgisApiHelper
+
+  let(:subject) { ArcgisApi::Geocoder.new }
+
+  describe '#suggest' do
```

Comment:
When testing methods that work with collections, I recommend including different cases for zero, one, and multiple results.